src/istio/control/tcp/attributes_builder_test.cc

```diff
@@ -372,12 +379,21 @@ TEST(AttributesBuilderTest, TestCheckAttributes) {
   EXPECT_CALL(mock_data, GetPrincipal(_, _))
       .WillRepeatedly(Invoke([](bool peer, std::string* user) -> bool {
         if (peer) {
-          *user = "test_user";
+          *user = "cluster.local/sa/test_user/ns/test_ns/";
         } else {
           *user = "destination_user";
         }
         return true;
       }));
+  EXPECT_CALL(mock_data, GetSourceNamespace(_, _))
```

Comment:
I don't think this mock method is called.

Reply:
Removed.